drivers/ng_at86rf2xx/ng_at86rf2xx_getset.c

```diff
@@ -27,14 +28,51 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+#ifdef MODULE_NG_AT86RF212B
+static const uint8_t dbm_to_tx_pow_868[] = {0x1d, 0x1c, 0x1b, 0x1a, 0x19, 0x18,
+                                        0x17, 0x15, 0x14, 0x13, 0x12, 0x11,
+                                        0x10, 0x0f, 0x31, 0x30, 0x2f, 0x94,
+                                        0x93, 0x91, 0x90, 0x29, 0x49, 0x48,
+                                        0x47, 0xad, 0xcd, 0xcc, 0xcb, 0xea,
+                                        0xe9, 0xe8, 0xe7, 0xe6, 0xe4, 0x80,
+                                        0xa0};
+static const uint8_t dbm_to_tx_pow_915[] = {0x1d, 0x1c, 0x1b, 0x1a, 0x19, 0x17,
+																			  0x16, 0x15, 0x14, 0x13, 0x12, 0x11,
+																			  0x10, 0x0f, 0x0e, 0x0d, 0x0c, 0x0b,
+																			  0x09, 0x91, 0x08, 0x07, 0x05, 0x27,
+																			  0x04, 0x03, 0x02, 0x01, 0x00, 0x86,
+																			  0x40, 0x84, 0x83, 0x82, 0x80, 0xc1,
+																			  0xc0};
```

Comment:
ups, did not scroll over to the right to see that there is actually content... One more reason to keep the indention clean :-)